manager-bundle/src/ContaoManager/Plugin.php

```diff
@@ -403,7 +425,8 @@ private function getDatabaseUrl(ContainerBuilder $container): string
         }
 
         return sprintf(
-            'mysql://%s%s:%s%s',
+            '%s://%s%s:%s%s',
+            $driver,
```

Comment:
```suggestion
            str_replace('_', '-', $driver),
```

See https://www.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/configuration.html#connecting-using-a-url : 

> The scheme names representing the drivers are either the regular driver names (see below) with any underscores in their name replaced with a hyphen (to make them legal in URL scheme names) …

Reply:
fixed in c3bd64236b7bec212f8758a0d5ef50a9915faa45